python/tests/integration/arcticdb/test_s3.py

```diff
@@ -66,3 +69,32 @@ def test_nfs_backed_s3_storage(lib_name, nfs_backed_s3_storage):
 
     for o in objects:
         assert re.match(bucketized_pattern, o.key), f"Object {o.key} does not match pattern {bucketized_pattern}"
+
+
+@pytest.fixture(scope="function", params=[MotoNfsBackedS3StorageFixtureFactory, MotoS3StorageFixtureFactory])
+def s3_storage_dots_in_path(request):
+    prefix = "some_path/.thing_with_a_dot/even.more.dots/end"
```

Comment:
How does this change interact with the fact that we convert the first `.` in lib names into a `/`?
e.g. The path for `aowens.test` would be `some_path/.snapshot/<snap name>/aowens/test`

Reply:
There are some notes here https://manwiki.maninvestments.com/x/QCAuFg . You just pass the whole path through `some_path/.snapshot/<snap name>/aowens/test`